### PR TITLE
Bumb Fiber v1.3.1

### PR DESCRIPTION
**Pull request by Fenny**

### CODE REVIEW

File: go/fiber/config.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 framework:
   website: gofiber.github.io/fiber
-  version: 1.2
+  version: 1.3.1
```
Comment (waghanza) [inline go/fiber/config.yaml]: could you specify `1.3` instead ?

File: go/fiber/go.mod [hunk 2]
```diff
@@ -1,3 +1,5 @@
 module main
 
-require github.com/gofiber/fiber v1.2.3
+require github.com/gofiber/fiber v1.3.1
+
+go 1.13
```
Comment (waghanza) [inline go/fiber/go.mod]: could you remove this line ?

File: go/fiber/go.sum [hunk 3]
```diff
@@ -0,0 +1,28 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```
Comment (waghanza) [inline go/fiber/go.sum]: could you remove this file ?

**Comment by Fenny** [comment]
@waghanza Thanks!

**Comment by waghanza** [comment]
@Fenny I merge too ealy

I have
~~~
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/gofiber/fiber.getRegex(0x8baa64, 0x9, 0xc00008b670, 0x1, 0x1)
	/go/pkg/mod/github.com/gofiber/fiber@v1.3.1/utils.go:38 +0x2c4
github.com/gofiber/fiber.(*Fiber).register(0xc0000b0840, 0x8b8fce, 0x3, 0xc000092f10, 0x2, 0x2)
	/go/pkg/mod/github.com/gofiber/fiber@v1.3.1/router.go:85 +0x570
github.com/gofiber/fiber.(*Fiber).Get(...)
	/go/pkg/mod/github.com/gofiber/fiber@v1.3.1/methods.go:59
main.main()
	/go/src/app/main.go:11 +0x140
~~~

when running compiled binary

**Comment by Fenny** [comment]
My mistake, fixed in https://github.com/the-benchmarker/web-frameworks/pull/2232

**Pull request merged**